test/Samples.MVCApp.Tests/Samples.MVCApp.Tests.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
+    <PackageReference Include="Microsoft.Playwright.NUnit" Version="1.25.0" />
+    <PackageReference Include="NUnit" Version="3.13.3" />
```

Comment:
We're using xunit elsewhere in the project. Is there a need to use NUnit?

Reply:
I just followed the playwright docs, I can see if they have xunit versions

Reply:
Doesn't look like there's as mature built in support for xunit currently, as there are lots of projects, i.e. https://github.com/martincostello/dotnet-playwright-tests which seem to be trying to address that integration, I think we should consider just using the playwright nunit (or mstest packages) rather than taking on the extra complexity of trying to make it work with xunit?  https://playwright.dev/dotnet/docs/test-runners#xunit-support

Reply:
> While Playwright for .NET isn't tied to a particular test runner or testing framework, in our experience it works best with the built-in .NET test runner, and using NUnit as the test framework. NUnit is also what we use internally for [our tests](https://github.com/microsoft/playwright-dotnet/tree/main/src/Playwright.Tests).

Reply:
That's reasonable. I'm not familiar with nunit, but I'm assuming it's got similar constructs :)